### ISSUE TITLE
Option to select whether individual holds a Master's degree

### ISSUE THREAD

**Issue by kyranjamie** [stevermeister/dutch-tax-income-calculator-old#28]
Great job mate! Liking the material & new domain.

Regarding 30% rule checkbox: it'd be great if, when selected, a toggle is displayed allowing you to chose whether or not a Master's degree is held – as stated in the explanation beneath, this changed the sum from which relief is applied.

**Comment by stevermeister** [comment]
thank you for feedback!

did not get your point about "Master's degree". do you want system to guess whether you have ruling or not?

**Comment by kyranjamie** [comment]
Ah sorry, let me try again!

The tax benefit is applied differently depending on education level, as mentioned in the footnote. Earnings from:
-  Without MSc: €36,889
-  With MSc: €28,041

Take the example of, say, a €60000 salary. The tax reduction would apply to income of:
-  Without MSC: €31,959
-  With MSc: €23,111

Given this, the user has to input whether they have a Master's to calculate net salary correctly – currently it is calculated as if a Master's is held.

http://www.iamsterdam.com/en/local/official-matters/highly-skilled-migrants/thirty-percent-ruling/30-percent-ruling-indepth

**Comment by stevermeister** [comment]
there is already such switcher (if I understand you correctly) and it's called "I have 30% ruling", no?

> Take the example of, say, a €60000 salary. The tax reduction would apply to income of:
> Without MSC: €31,959
> With MSc: €23,111

from where did you get this calculations? may be it's something wrong with mine.

**Comment by kyranjamie** [comment]
Yes, the option to select 30% rule is there. However, it is important to note you _do not_ need a Master's to be eligible for it.

Take the scenario of two people earning 60k, who _both_ have 30% ruling. One **has** a Master's degree, the other **does not**.

[This source](http://www.iamsterdam.com/en/local/official-matters/highly-skilled-migrants/thirty-percent-ruling/30-percent-ruling-indepth#conditions) explains the difference better than I can, but
if you don't have a MSc, you must earn **at least** €36889 
If you do, the minimum amount is lower, only: €28,041

**Has MSc**: €60000 - €28041 = €31959
**No MSc**: €60000 - €36889 = €23111

We calculate the earnings on which the ruling can be applied, by subtracting the minimum required from the gross salary.  This is why it's necessary to state education level.

**Comment by yevgeniyvaleyev** [comment]
This system doesn't give you an option to guess if you are eligible for ruling. 

It uses your CURRENT state. If you already have ruling you just switch that option on. 

Decision about ruling is taken by tax department (Belastingdienst). So adding this option you want could lead to misunderstanding for people who calculate their nett salary. 

We could add "?" link to Belastingdienst website, so if you are interested in ruling but still don't have it, you could investigate this possibility for you personally.

**Comment by kyranjamie** [comment]
Sure, adding more information would indeed be a good idea!

Perhaps I've miscommunicated this though. I am not suggesting we add a `guess whether eligible or not` option. This would not make sense.

The **calculation of the net salary with 30% rule is different depending on education status**. Currently, it makes the assumption you hold an MSc.

The two different conditions are explained in the about section:
![image](https://cloud.githubusercontent.com/assets/1618764/12720101/2c1b73e6-c8f9-11e5-9cf6-fbfc2d200a14.png)

Mocked it up here:
![image](https://cloud.githubusercontent.com/assets/1618764/12720215/e06ff4fc-c8f9-11e5-882e-dc8dd2bbc1a8.png)

Checking Master's degree box ===  bigger net income

**Comment by asizikov** [comment]
@kyranjamie I'm afraid you misunderstood this information. The text on your screenshot says about the minimum salary which one must have to be eligible. 
If you have master degree and your taxable income is 37k you'll get the same amount as the person with bachelor degree earning 37k. But the person without master degree will not get 30% if she earns less than 36,7k.

**Comment by kyranjamie** [comment]
@asizikov Thanks, quick reply!

Hopefully you can elaborate on my misunderstanding. 

This information states the **minimum taxable salary** for 2015. The 30% rules allows `up to` 30% of your salary to be paid tax-free. (However it will be a lower percentage **if** 30% tax free brings you under the minimum taxable salary limit.)

Let's use your example of 37k, where both people qualify (the guy with a Bachelor's only just!).

| Education level | Min. taxable salary (2016) | Taxable amount | Tax free salary (%)* |
| --- | --- | --- | --- |
| Master's | €28041 | €8959 | 24.2% |
| Bachelor's | €36889 | €111 | 0.03% |

*What percent, up to 30, can be paid tax free **without going under minimum taxable salary**

What would be the point of having a different qualification bracket for someone with a Master's, if you then earn the same as someone with a Bachelors?

**Comment by asizikov** [comment]
Person A. Earns 40000. When discount applied the taxable income becomes 28,000. If A has master degree and she is under 30 years old she is eligible for the discount.

Person B. Earns 40k. No master degree or/and over 30 y.o. Taxable income ls less then 36k. This person is not eligible. But might get a lower percentage.

Person C. Earns 54k. Taxable income is 37.8. This person is eligible regardless of the age and degree.

This is how tax office decides to gain this discount. Once you have the discount you can use this calculator to find out your net income.

**Comment by kyranjamie** [comment]
Person A, B and C all qualify for the ruling.

Why do you say Person B is not eligible? Qualification itself is binary. So, in order to get a lower percentage, they **must** qualify.

Person B does however earn net less than Person A, because of the taxable salary minimum difference. This is the what _could_ be calculated, were education level known.

Despite the non-qualification of Person B, I'm not sure how the information you've shared disagrees with what I've stated, or where the misunderstanding lies?

All the same, I guess it's not too important, since it only benefits those with both no Master's **and** the 30% rule.

**Comment by pvinis** [comment]
i was looking for that option, because i also understood that with a masters, one would earn net more. after reading the conversation, i still dont know if my and @kyranjamie's understanding is correct or not. i guess we need someone who knows about the dutch tax system.

**Comment by omcgo** [comment]
I know _about_ the Dutch tax system and I agree with @kyranjamie & @pvinis.

**Comment by stevermeister** [comment]
@omcgo you are more than welcome to make a PR :)

**Comment by pvinis** [comment]
im waiting to get my 30%, and i added that i have a master. when i have it, i will see and hopefully understand and make a pr. i also have to remember!

**Comment by omcgo** [comment]
Best of luck :)
